Update faq.md (#4613)
kubernetes-sigs/kubebuilder: docs/book/src/faq.md

</aside>

> ## How is the value informed via the domain flag (i.e. `kubebuilder init --domain example.com`) when we init a project?

Reviewer: ```suggestion
## How is the value set via the domain flag (`kubebuilder init --domain example.com`) used when initialising a project?
```

Reviewer: Maybe ?
